README.md

```diff
@@ -31,6 +31,8 @@ Hugo can be installed by following
 
 Hugo (extended) releases can be downloaded from
 [the Hugo releases page](https://github.com/gohugoio/hugo/releases).
+The recommended version of Hugo to use is
+[v0.110.0](https://github.com/gohugoio/hugo/releases/tag/v0.110.0).
```

Comment:
I wonder if it makes sense to reference the exact version here, while it can be taken from the configuration file.
You could just point to that file saying it contains the currently supported/used version.

Reply:
Good idea, thanks! It's been added now.

Reply:
Great, thanks!